### PR TITLE
Bump nixpkgs to 22.11

### PR DESCRIPTION
**Pull request by lheckemann**
Allows removing some patches!

### CODE REVIEW

**Comment by cole-h** [comment]
FYI: 22.11 is no longer beta :) https://github.com/NixOS/nixos-org-configurations/pull/228

File: flake.nix [hunk 1]
```diff
@@ -127,13 +127,9 @@
           });
 
         propagatedDeps =
-          [ ((boehmgc.override {
+          [ (boehmgc.override {
               enableLargeConfig = true;
-            }).overrideAttrs(o: {
-              patches = (o.patches or []) ++ [
-                ./boehmgc-coroutine-sp-fallback.diff
```
Comment (bb010g) [inline flake.nix:134]: `./boehmgc-coroutine-sp-fallback.diff` is no longer used & can be deleted.

**Review by bb010g — APPROVED** [review]

**Comment by edolstra** [comment]
@lheckemann Looks like this broke some of the VM tests, e.g. https://hydra.nixos.org/build/203389412. Can you have a look at that? I guess we need some more packages in `virtualisation.additionalPaths`.

**Comment by lheckemann** [comment]
https://github.com/NixOS/nix/pull/7570 fixes them

File: flake.nix (hunk 1 above)
Comment (roberth) [reply flake.nix:134]: What? How? Our `BoehmGCStackAllocator` solution does not work correctly with vanilla boehmgc!

**Pull request merged**